models/product.model.lkml

```diff
@@ -78,3 +78,21 @@ explore:fct_board_activity {
     sql_on: ${fct_board_activity.server_id} = ${dim_board_customers.server_id} ;;
   }
 }
+
+explore:fct_calls_activity {
+  label: "Telemetry Active Calls Servers"
+  group_label: "[New] Active Servers"
+
+
+  join: dim_server_info {
+    relationship: many_to_one
+    type: full_outer
```

Comment:
Is there a check that ensures every row in `fct_calls_activity` has a matching row in `dim_server_info`?

Reply:
Added this check here https://github.com/mattermost/mattermost-data-warehouse/pull/1357